### ISSUE TITLE
Do you try 512 resolution？

### ISSUE THREAD

**Issue by hudongloop** [SeitaroShinagawa/chainer-partial_convolution_image_inpainting#3]
Hi，
I'm also try to implement this paper by pyTorch. Input 512x512 image (celeA-HQ) and 2 batch_size. But the result have much watermark in the hole edge and low-resolution in generate hole area. Do you have this situation?
Thank you!

**Comment by SeitaroShinagawa** [comment]
Hi @hudongloop,
Thanks for your attention! Actually, I have never tried the situation. I will try it soon.  

To try your situation, I will download celebA-HQ and VGG face pre-trained model from the following links:   
celebA-HQ: https://github.com/nperraud/download-celebA-HQ  
VGG face pre-trained model: http://www.robots.ox.ac.uk/~vgg/software/vgg_face/  
Is it OK? I'm glad if you give me some advice.   

I will also try 512x512 situation with place2 dataset. Thank you!

**Comment by hudongloop** [comment]
Hi @SeitaroShinagawa,
Yes, the dataset and VGG pre-trained model just as you say。But I use convolution feature(conv1-2, conv2-2, conv3-3) for perceptual loss and style loss. And the masks are generated according to the method in this paper (have discrepancy). Would you know how the method of the mask is generated in this paper.
I would like to know the result of your 256x256 have some watermark？

Thank you for your quick reply.

**Comment by SeitaroShinagawa** [comment]
Hi @hudongloop,
The follwing image is the generated examples from place2 validation set (but masks are from trainset).  
Model is trained with 500,000 iterations (without fine-tuning stage). 

The image from left column shows mask, I_out, I_comp, I_gt, respectively.  
Judging from it, I can not find apparent watermark. However, I used masks generated with a different way (see [generate_windows.py](https://github.com/SeitaroShinagawa/chainer-partial_convolution_image_inpainting/blob/master/generate_windows.py)).  
I'm not sure whether my code works with same mask dataset to original paper.  

Best,  

![generated](https://user-images.githubusercontent.com/7993233/40272644-9e4dd42c-5beb-11e8-9d48-f6aa91eb74f0.jpg)
